src/types/Template.ts

```diff
@@ -0,0 +1,11 @@
+import type { UserRef } from './User';
+
+export interface Template {
+    id: string;
+    title: string;
+    content: string;
+    creator: UserRef | null;
+    last_modifying_user: UserRef | null;
+    created_at: string;
```

Comment:
Can you use the `Iso8601DateTime` common type here instead of just `string`? Same for `modified_at`.